src/fix/rgbfix.1

```diff
@@ -152,13 +168,11 @@ a valid size:
 The following will make a SGB-enabled, color-enabled game with a title of
 .Dq foobar ,
 and pad it to a valid size.
-.Po
-The Game Boy itself does not use the title, but some emulators or ROM managers do.
-.Pc
+.Pq The Game Boy itself does not use the title, but some emulators or ROM managers do.
 .Pp
 .D1 $ rgbfix -vcs -l 0x33 -p 255 -t foobar baz.gb
 .Pp
-The following will duplicate the header (sans global checksum) of the game
+The following will duplicate the header, sans global checksum, of the game
```

Comment:
"The following will duplicate the header of the game, sans global checksum"

Reply:
It's in the continuity of "SURVIVAL KIDS", so it's meant to read

> The following will duplicate the header, sans global checksum, of the game `SURVIVAL KIDS`:

Is that good enough?

Reply:
Better, but how about "The following will duplicate the header of the game `SURVIVAL KIDS`, sans global checksum:"?